website/components/Head/index.js

```diff
@@ -0,0 +1,33 @@
+import NextHead from 'next/head';
+import PropTypes from 'prop-types';
+
+const Head = ({ meta }) => {
+  return (
+    <NextHead>
+      <title>{meta?.title}</title>
+      <meta content={meta?.description} name="description" />
+      <meta property="og:type" content={meta?.type} />
+      <meta property="og:site_name" content="Strapi Design System" />
+      <meta property="og:description" content={meta?.description} />
+      <meta property="og:title" content={meta?.title} />
```

Comment:
Can you add `<html lang="en" />` please

Reply:
You forgot this comment.

Reply:
I updated the _document.js <html lang="en" /> on nextjs, don't want to have this <HTML lang="en" /> in the Head